src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    date={article.date}
+    text={article.text}
```

Comment:
```suggestion
    {...article}
```
It's correct, but here is another way to do it

https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493